relay-general/src/store/transactions/processor.rs

```diff
@@ -2836,6 +2835,13 @@ mod tests {
         "select count() from table where id in (%s)"
     );
 
+    span_description_test!(
+        span_description_scrub_mixed,
+        "UPDATE foo SET a = %s, b = log(e + 5) * 600 + 12345 WHERE true",
+        "db.sql.query",
+        "UPDATE foo SET a = %s, b = log(e + %s) * %s + %s WHERE %s"
```

Comment:
This was taken from a real-world example. If the incoming query is only partially parameterized, IMO we should still attempt to scrub all values from the query.